releasetasks/templates/firefox/l10n_changesets.yml.tmpl

```diff
@@ -0,0 +1,103 @@
+{% set buildername = "{}_l10n_changeset".format(branch) %}
+{% set buildername_beet = "{}_beet".format(buildername) %}
+{% do push_to_releases_extra_upstream_builders.append(buildername_beet) %}
```

Comment:
Hmmm, not sure... Probably we shouldn't, like any other txt/json file. I'l remove this dependency.